api/src/test/java/org/openmrs/api/ConceptServiceTest.java

```diff
@@ -182,6 +182,7 @@ public void saveConcept_shouldSaveAConceptNumericAsAConcept() throws Exception {
 		Concept c2 = new Concept(2);
 		ConceptName cn = new ConceptName("not a numeric anymore", Locale.US);
 		c2.addName(cn);
+		c2.addDescription(new ConceptDescription("some description",Context.getLocale()));
```

Comment:
You no longer need to set this locale here since the save handler does this for you